### PR TITLE
Generate 1 5 clientset

### PR DESCRIPTION
**Pull request by caesarxuchao**
Generate the 1.5 clientset. Stop updating 1.4 clientset. Remove 1.2 clientset.

@nikhiljindal @lavalamp 

I will rebase #31994 atop of this one.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32243)

### CODE REVIEW

**Comment by caesarxuchao** [comment]
@nikhiljindal I removed the commit that touches federation.

File: test/e2e/framework/framework.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ type Framework struct {
 	BaseName string
 
 	Client        *client.Client
-	Clientset_1_2 *release_1_2.Clientset
+	Clientset_1_5 *release_1_5.Clientset
 	Clientset_1_3 *release_1_3.Clientset
```
Comment (nikhiljindal) [inline test/e2e/framework/framework.go]: Also delete Clientset_1_3 and StagingClient and replace their usages by 1.5 clientset?

**Comment by nikhiljindal** [comment]
Thanks, LG

File: test/e2e/framework/framework.go (hunk 1 above)
Comment (caesarxuchao) [inline test/e2e/framework/framework.go]: I'll remove Clientset_1_3 but keep StagingClient (it's using "k8s.io/client-go/1.4/kubernetes")

**Comment by caesarxuchao** [comment]
Some trivial fixes in e2e test. Adding back the lgtm.

**Comment by caesarxuchao** [comment]
@k8s-bot test this, issue #IGNORE

**Comment by caesarxuchao** [comment]
Changed a link in CHANGELOG.md to not point to the removed release_1_2 clientset.

**Comment by caesarxuchao** [comment]
Just ran hack/update-munge-docs.sh, refresh the lgtm myself.

**Pull request merged**